docs/source/tracking.rst

```diff
@@ -718,11 +718,14 @@ Azure Blob Storage
 To store artifacts in Azure Blob Storage, specify a URI of the form
 ``wasbs://<container>@<storage-account>.blob.core.windows.net/<path>``.
 MLflow expects Azure Storage access credentials in the
-``AZURE_STORAGE_CONNECTION_STRING`` or ``AZURE_STORAGE_ACCESS_KEY`` environment variables (preferring
-a connection string if one is set), so you must set one of these variables on both your client
-application and your MLflow tracking server. Finally, you must run ``pip install azure-storage-blob``
-separately (on both your client and the server) to access Azure Blob Storage; MLflow does not declare
-a dependency on this package by default.
+``AZURE_STORAGE_CONNECTION_STRING``, ``AZURE_STORAGE_ACCESS_KEY`` or all three of
+``AZURE_TENANT_ID`` & ``AZURE_CLIENT_ID`` & ``AZURE_CLIENT_SECRET`` environment variables
+(preferring a connection string over the access key, and the access key over the client credentials),
+so you must set one of these variables on both your client application and your MLflow tracking server.
+Finally, you must run ``pip install azure-storage-blob`` separately (on both your client and the server)
```

Comment:
unfortunately this is no longer true as of [this commit](https://github.com/mlflow/mlflow/commit/74be64c8c469aa6cce5cebb92fcafea4bd5ad3a9#diff-60f61ab7a8d1910d86d9fda2261620314edcae5894d5aaa236b821c7256badd7), so ...